test/ngAnimate/animateRunnerSpec.js

```diff
@@ -0,0 +1,251 @@
+'use strict';
+
+describe("$$rAFMutex", function() {
```

Comment:
No, no, no, no !
No !
No, no annnd no !

Consistent use of `'`, please :stuck_out_tongue: